instrumentation/spring/spring-boot-autoconfigure/src/main/java/io/opentelemetry/instrumentation/spring/autoconfigure/internal/properties/SpringConfigProperties.java

```diff
@@ -29,37 +31,87 @@ public class SpringConfigProperties implements ConfigProperties {
   private final ExpressionParser parser;
   private final OtlpExporterProperties otlpExporterProperties;
   private final OtelResourceProperties resourceProperties;
-  private final PropagationProperties propagationProperties;
+  private final SpringOtelProperties springOtelProperties;
```

Comment:
How about using `OtelSpringProperties` here? It uses same naming convention with other variables like `OtelResourceProperties`.